docs/docs/icechunk-python/virtual.md

```diff
@@ -209,4 +191,4 @@ No extra configuration is necessary for local filesystem references.
 
 ### Virtual Reference File Format Support
 
-Currently, Icechunk supports `HDF5` and `netcdf4` files for use in virtual references. See the [tracking issue](https://github.com/earth-mover/icechunk/issues/197) for more info.
+Currently, Icechunk supports `HDF5` and `netcdf4` files for use in virtual references with `VirtualiZarr`. Support for other filetypes such as TIFF or GRIB2 are on the roadmap.
```

Comment:
This capability is completely independent of Icechunk isn't it? As soon as those formats are supported in VirtualiZarr they will immediately work in icechunk? So maybe this should make that clear and link to virtualizarr issues?